Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/IMCAbsorptionStrategy.h

```diff
@@ -0,0 +1,48 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAPI/ISpectrum.h"
+#include "MantidAlgorithms/DllConfig.h"
+#include "MantidAlgorithms/SampleCorrections/IMCInteractionVolume.h"
+#include "MantidAlgorithms/SampleCorrections/MCInteractionStatistics.h"
+#include "MantidHistogramData/Histogram.h"
+#include "MantidKernel/DeltaEMode.h"
+#include <tuple>
+
+namespace Mantid {
+namespace API {
+class Sample;
+}
```

Comment:
Does this need a comment here to match the others?
```suggestion
} // namespace API
```